test/integrated/soil_energy_hydrology_biogeochemistry.jl

```diff
@@ -1,5 +1,6 @@
 using Test
 using ClimaCore
+import ClimaParams
```

Comment:
do we need this?

Reply:
I think we somehow have some leakages in our tests with the environments? because this runs in CI without this change, but if I just try and run it locally by itself, it fails because it doesnt have ClimaParams.